lib/Mojolicious.pm

```diff
@@ -41,14 +42,29 @@ has plugins            => sub { Mojolicious::Plugins->new };
 has preload_namespaces => sub { [] };
 has renderer           => sub { Mojolicious::Renderer->new };
 has routes             => sub { Mojolicious::Routes->new };
+has secrets_file       => sub { $ENV{MOJO_SECRETS_FILE} || shift->home->rel_file('mojo.secret') };
 has secrets            => sub {
   my $self = shift;
+  my $file = $self->secrets_file;
 
-  # Warn developers about insecure default
-  $self->log->trace('Your secret passphrase needs to be changed (see FAQ for more)');
+  if (-f $file) {
+    my @secrets = grep { length $_ >= 22 }    # 22 random chars in base62 alphabet should be at
+                                              # best 128 bits of entropy
```

Comment:
Nitpick we like to keep comments aboce the code, not behind. And i think some other comments use unnecessary punctuation.

Reply:
Fixed